microstream/build.gradle.kts

```diff
@@ -3,6 +3,7 @@ plugins {
 }
 
 dependencies {
+    api(libs.micronaut.aop)
```

Comment:
should this be implementation instead?

Reply:
🤔  If a user has this module as a dependency, it will _*require*_ AOP to be pulled in as well won't it?  I'm not 100% here whether it should be api or implementation 🤔  @melix

Reply:
I've changed to implementation btw 👍

Reply:
The difference between `api` and `implementation` is not whether it's pulled in transitively or not, but _when_ it's transitively pulled in. With `api`, it's pulled in when the consumer is compiled, which means that the dependency is exposed in the API of the component. When it's `implementation`, it's only exposed at runtime. The choice between the 2 therefore only depends on whether the dependency is exposed in the API or not.

Reply:
Cool, thanks @melix `implementation` it is then!

Good luck tomorrow!